.gitmodules

```diff
@@ -43,7 +43,7 @@
 	url = https://github.com/ceph/dmclock.git
 [submodule "src/seastar"]
 	path = src/seastar
-	url = https://github.com/ceph/seastar.git
+	url = https://github.com/leonid-s-usov/ceph-seastar.git
```

Comment:
I'd suggest that this prevents this PR from being merged for now.
@rzarzynski ?

Reply:
I am not expecting this PR to be merged until those mentioned PRs are merged or rejected, and this PR gets updated with the resulting submodule commits back at the original URL.

Until then, I had to change the URLs so that people who can test this (like @vshankar) could simply checkout the branch.

Reply:
Just to make sure you know that this link is still there

Reply:
https://github.com/ceph/seastar/pull/14
https://github.com/ceph/libkmip/pull/1
are still pending review. I've commented on those.

s3select was merged and I've rolled back the URL and updated the submodule commit.

Reply:
Unresolving this thread. Just for local testing – sure; though, before merging, we should incorporate fixes (for Mac, I assume) to `ceph/seastar.git` or – preferably – merge with the upstream's seastar repo.

Reply:
@athanatos What's the right way to update Seastar? Is it possible that upstream Seastar doesn't need these patches and we can just update to current?